### ISSUE TITLE
Hello,I need your help.How do you get the edge label Edge in the training set?I look forward to your reply. Thanks!

### ISSUE THREAD

**Issue by yike5022** [GewelsJI/VPS#28]

**Comment by GewelsJI** [comment]
This code is not hard to code, and you can try Matlab version (https://github.com/JXingZhao/EGNet/blob/master/sal2edge.m) or rewrite them in a pytorch way.

**Comment by yike5022** [comment]
Thank you for your reply!
